schemes/musica/musica_ccpp.meta

```diff
@@ -147,23 +141,17 @@
   units = None
   dimensions = ()
   intent = in
-[ number_of_photolysis_wavelength_grid_sections ]
-  standard_name = number_of_photolysis_wavelength_grid_sections
-  type = integer
-  units = None
-  dimensions = ()
-  intent = in
 [ photolysis_wavelength_grid_interfaces ]
   standard_name = photolysis_wavelength_grid_interfaces
   type = real | kind = kind_phys
-  units = nm
-  dimensions = (horizontal_loop_extent)
+  units = m
+  dimensions = (photolysis_wavelength_grid_interface_dimension)
   intent = in
 [ extraterrestrial_flux ]
   standard_name = extraterrestrial_radiation_flux
   type = real | kind = kind_phys
-  units = photons cm-2 s-1 nm-1
-  dimensions = (horizontal_loop_extent)
+  units = count cm-2 s-1 nm-1
```

Comment:
After talking with Michael yesterday it seemed like `photons` wouldn't be a valid unit and `count` was an existing unit that seemed appropriate, but I could be thinking about this wrong. @peverwhee or @nusbaume - do you have any recommendations on choosing units?

Reply:
i'd vote for `cm-2 s-1 nm-1` (no count), but units aren't exactly my forte.

Reply:
I think @peverwhee's suggestion is best if we include the word `photon` somewhere in the standard name.  Otherwise I would probably choose `photons cm-2 s-1 nm-1` over `count cm-2 s-1 nm-1`, as `count` is usually reserved for dimension variables (e.g. `vertical_layer_dimension`).  Anyways, that's my two cents!

Reply:
ah, ok - so is `photons cm-2 s-1 nm-1` a valid option? If so, I might lean in that direction

Reply:
I don't see it listed in the dictionary yet, but I feel like that makes it clear as to what the "flux" actually is.  If for some reason the other CCPP folks object to it then we can always fix it later to whatever is agreed upon.